src/runtime-rs/crates/shim/src/shim_run.rs

```diff
@@ -45,6 +48,9 @@ impl ShimExecutor {
         info!(sl!(), "start to run");
         self.args.validate(false).context("validate")?;
 
+        // Regist service logger for later use
+        logging::regist_subsystem_logger("runtimes", "service");
```

Comment:
Nit
suggest changing this to `register_subsystem_logger`, same as the comment above